src/WebJobs.Script/Diagnostics/FunctionFileLoggerProvider.cs

```diff
@@ -50,7 +50,7 @@ public ILogger CreateLogger(string categoryName)
             {
                 // Make sure that we return the same fileWriter if multiple loggers write to the same path. This happens
                 // with Function logs as Function.{FunctionName} and Function.{FunctionName}.User both go to the same file.
-                IFileWriter fileWriter = _fileWriterCache.GetOrAdd(filePath, (p) => _fileWriterFactory.Create(Path.Combine(_roogLogPath, filePath)));
+                IFileWriter fileWriter = _fileWriterCache.GetOrAdd(filePath, (path) => _fileWriterFactory.Create(Path.Combine(_roogLogPath, path)));
```

Comment:
Note: this pulls in `_roogLogPath` and could be a local function to be more intuitive from the instance if we want - can fix the typo at the same time. Was aiming at minimal changes but happy to include here.